CONTRIBUTING.md

```diff
@@ -114,6 +114,30 @@ If for any reason there is an update to the build tools, you will then need to f
 3. Invoke `dotnet test eng\service.proj --filter TestCategory!=Live`
    <br/><br/>
 
+### Live testing
+
+Live tests assume a live resource has been created and appropriate environment
+variables have been set for the test process. To automate setting up live
+resources we use created a script called `New-TestResources.ps1` that deploys
+resources for a given service.
+
+To see what resources will be deployed for a live service, check the
+`test-resources.json` ARM template files in the service you wish to deploy for
+testing.
```

Comment:
You might give an example here, like "testing, for example: sdk\keyvault\test-resources.json` (I suggest mine as an example only since I wrote it first to test my scripts and know it works well).